frontend/src/pages/modelServing/screens/projects/NIMServiceModal/DeployNIMServiceModal.tsx

```diff
@@ -0,0 +1,327 @@
+
+  const baseInputValueValid =
+    createDataServingRuntime.numReplicas >= 0 &&
+    resourcesArePositive(createDataServingRuntime.modelSize.resources) &&
+    requestsUnderLimits(createDataServingRuntime.modelSize.resources);
+
+  const isDisabledInferenceService =
+    actionInProgress ||
+    createDataInferenceService.name.trim() === '' ||
+    createDataInferenceService.project === '' ||
+    !isInferenceServiceNameWithinLimit ||
+    !baseInputValueValid;
+
+  const servingRuntimeSelected = React.useMemo(
+    () =>
+      editInfo?.servingRuntimeEditInfo?.servingRuntime ||
+      getServingRuntimeFromName('nvidia-nim-runtime', servingRuntimeTemplates),
+    [editInfo, servingRuntimeTemplates],
+  );
+
+  const onBeforeClose = (submitted: boolean) => {
+    onClose(submitted);
+    setError(undefined);
+    setActionInProgress(false);
+    resetDataServingRuntime();
+    resetDataInferenceService();
+    resetSelectedAcceleratorProfile();
+    setAlertVisible(true);
+  };
+
+  const setErrorModal = (e: Error) => {
+    setError(e);
+    setActionInProgress(false);
+  };
+
+  const onSuccess = () => {
+    setActionInProgress(false);
+    onBeforeClose(true);
+  };
+
+  const submit = () => {
+    setError(undefined);
+    setActionInProgress(true);
+
+    const servingRuntimeName =
+      editInfo?.inferenceServiceEditInfo?.spec.predictor.model?.runtime ||
+      translateDisplayNameForK8s(createDataInferenceService.name);
+
+    const submitServingRuntimeResources = getSubmitServingRuntimeResourcesFn(
+      servingRuntimeSelected,
+      createDataServingRuntime,
+      customServingRuntimesEnabled,
+      namespace,
+      editInfo?.servingRuntimeEditInfo,
+      false,
+      acceleratorProfileState,
+      selectedAcceleratorProfile,
+      NamespaceApplicationCase.KSERVE_PROMOTION,
+      projectContext?.currentProject,
+      servingRuntimeName,
+      true,
+    );
+
+    const submitInferenceServiceResource = getSubmitInferenceServiceResourceFn(
+      createDataInferenceService,
+      editInfo?.inferenceServiceEditInfo,
+      servingRuntimeName,
+      false,
+      acceleratorProfileState,
+      selectedAcceleratorProfile,
+      allowCreate,
+      editInfo?.secrets,
+      false,
+    );
+
+    Promise.all([
+      createNIMSecret(namespace, NIM_SECRET_NAME, false, false, dashboardNamespace),
+      createNIMSecret(namespace, NIM_NGC_SECRET_NAME, true, false, dashboardNamespace),
+      createNIMPVC(namespace, NIM_PVC_NAME, pvcSize, false),
+      submitServingRuntimeResources({ dryRun: true }),
+      submitInferenceServiceResource({ dryRun: true }),
+    ])
+      .then(() =>
+        Promise.all([
+          submitServingRuntimeResources({ dryRun: false }),
+          submitInferenceServiceResource({ dryRun: false }),
+        ]),
+      )
+      .then(() => onSuccess())
+      .catch((e) => {
+        setErrorModal(e);
+      });
+  };
+
+  return (
+    <Modal
```

Comment:
NIM Modal shows this:
![image](https://github.com/user-attachments/assets/9e82cb6c-a517-4b6b-99e8-8ef2af22784c)

KServe Modal shows this:
![image](https://github.com/user-attachments/assets/79ef062c-61ba-4e83-a79b-952534cb8189)


Not sure what's happening here, it looks like you added the same section 🤔

Reply:
Is this section not supposed to be there?

Reply:
Can you investigate why the spacing is not between the fields on your modal? Something is up... I don't have a clear understanding right away why KServe modal has spacing and yours doesn't (before the Accelerator file)

Reply:
Nah -- it's an existing problem; I'm just looking at new ones going forward.